### PR TITLE
Feature production env

### PR DESCRIPTION
**Pull request by edisinovcic**
### What does it do?
This PR introduces docker-compose for testing and production environments. 

#### List of deployed services
[UI](https://stagingverification.komputing.org/)
[Repository](https://stagingverification.komputing.org/repository/) - http://178.19.221.38:10000/repository/
[Server](https://stagingverification.komputing.org/server/health) -> check health of the server

Complete environment setup has been changed and that has to be updated in tests also.
Should registry images be moved to the Ethereum organization? @chriseth 
It can also be where it is. It's not a problem for this stage.

## TODO:
Finish apache config to solve all the routing problem

### Docker hub registry
Docker registry contains images for each of the services so people running services don't need to have whole source code, just docker-compose file. Services are located on these links:
[UI](https://hub.docker.com/repository/docker/blockchainit/source-verify-ui)
[Monitor](https://hub.docker.com/repository/docker/blockchainit/source-verify-monitor)
[Server](https://hub.docker.com/repository/docker/blockchainit/source-verify-server)
[Localchain](https://hub.docker.com/repository/docker/blockchainit/source-verify-localchain)
[Repository](https://hub.docker.com/repository/docker/blockchainit/source-verify-repository)

### CODE REVIEW

**Comment by edisinovcic** [comment]
@cgewecke please test that docker-compose-test is working for you.

Command to start it:
`docker-compose -f docker-compose-testing.yaml up --build`

**Comment by edisinovcic** [comment]
This PR only needs polishing of the apache config. Everything else seems working and ready.

**Comment by chriseth** [comment]
I do not feel qualified to review this. About the docker images: Yes, we can add it to the ethereum namespace. Would this require frequent updates and thus permissions?

**Comment by edisinovcic** [comment]
@chriseth we can add it to the ci to update docker images as well as npm so it should be completely automatic.

**Comment by chriseth** [comment]
If possible, I would like to avoid CI systems to have permission to update docker images.

**Comment by edisinovcic** [comment]
Okay, I can then create a bash script that need to be run manually if that's okay?

File: build_and_publish_docker_images.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+docker-compose -f docker-compose-build.yaml --no-cache --parallel
```
Comment (edisinovcic) [inline build_and_publish_docker_images.sh]: @chriseth would something like this be okay for building and pushing of docker images? 
Basically you have to be logged in locally to be able to publish it to docker hub.

**Comment by edisinovcic** [comment]
As this is current staging branch, will close this PR here.

Base for https://github.com/ethereum/source-verify/tree/staging

**Pull request closed without merging**